functions/firestore/users/onDelete.f.js

```diff
@@ -0,0 +1,13 @@
+import { firestore } from 'firebase-functions';
+import admin from 'firebase-admin';
```

Comment:
Does this need `require` or does it work fine like this? I've seen that bug before too, it's so strange.

Reply:
Yup, it's really odd, I think it has something to do with support for ES6 vs ES5 (`require` is for ES5?)